src/test/resources/data/AutomatedSessionRemindersTest.json

```diff
@@ -22,7 +22,7 @@
       "googleId": "AutSessRem.instructor",
       "courseId": "AutSessRem.course",
       "name": "Test Ins for Aut Sessions Reminder",
-      "email": "${support.email}",
+      "email": "${test.instructor}@gmail.com",
```

Comment:
This change doesn't look right. Either the original was wrong or the new one is wrong. From what I remember, we had a concept of 'support email address'. This is the email address that will be given in the TEAMMATES emails as the contact point, and probably as the 'reply to' address. Did we get rid of that concept at some point? If we did, we shouldn't have. If we didn't, then why was support email was used here originally and now it is being changed to test instructor's email?